.github/workflows/ci.yml

```diff
@@ -8,4 +8,7 @@ on:
 
 jobs:
   build:
-    uses: axonivy-market/github-workflows/.github/workflows/ci.yml@v3
+    uses: axonivy-market/github-workflows/.github/workflows/ci.yml@v4
+    with:
+      mvnArgs: '-Dmaven.test.failure.ignore=true'
```

Comment:
we may leave it like this for now; but I fear there is a real issue to address here.

Reply:
I removed `mvnArgs: '-Dmaven.test.failure.ignore=true'` to track and address the real issue.